z390test/src/test/groovy/org/z390/test/RunAssist.groovy

```diff
@@ -0,0 +1,48 @@
+package org.z390.test
+
+import org.junit.jupiter.api.Test
+
+
+class RunAssist extends z390Test {
+
+    var options = ['ASSIST', 'TRACEALL', "SYSMAC(${basePath("mac")})", "SYSOBJ(${basePath("linklib")})"]
+
+    @Test
+    void test_DEMOAST1() {
+        this.env.put('XPRNT', basePath('assist', 'demo', 'DEMOAST1.XPR'))
+        int rc = this.asmlg(basePath("assist", "demo", "DEMOAST1"), *options)
+        this.printOutput()
+        assert rc == 0
+    }
+
+    @Test
+    void test_TESTINS1() {
```

Comment:
"test_TESTINS1" should be "test_TESTAST1".

Reply:
Will fix